src/test/java/racingcar/util/ValidatorTest.java

```diff
@@ -0,0 +1,89 @@
+package racingcar.util;
+
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class ValidatorTest {
+
+    @Test
+    @DisplayName("자동차 이름이 5글자를 넘으면 예외 발생")
+    void carNameLengthTest() {
+        // 5글자 이상의 이름을 포함한 리스트로 검증
+        assertThatThrownBy(() -> Validator.carNamesValidate(List.of("pobi", "polar", "ihavelongname")))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
```

Comment:
에러 메세지 검증도 함께 있다면 좀 더 정확한 테스트가 될 수 있겠군요.

```suggestion
    @Test
    @DisplayName("자동차 이름이 5글자를 넘으면 예외 발생")
    void carNameLengthTest() {
        // 5글자 이상의 이름을 포함한 리스트로 검증
        assertThatThrownBy(() -> Validator.carNamesValidate(List.of("pobi", "polar", "ihavelongname")))
                .isInstanceOf(IllegalArgumentException.class);
                .hasMessage(ERR_INVALID_CAR_NAME_LENGTH);
    }
```

Reply:
출력되는 메시지까지는 검증을 해볼 생각을 생각을 못해봤네요 좋은 조언 감사합니다!